docs/src/development/local/_index.md

```diff
@@ -1,151 +1,92 @@
-  Receiving objects: 100% (3318/3318), 20.23 MiB | 3.71 MiB/s, done.
-  Resolving deltas: 100% (568/568), done.
-
-The project [project-id](abcdefgh1234567) was successfully downloaded to: 'project-directory'
-```
-
-You can now access your app's code from the project directory created on your computer.
-Note that inside the project directory, the `.platform/local` subdirectory is excluded from Git. 
-It contains builds and any local metadata about your project the [Platform.sh CLI](../../administration/cli/_index.md) needs.
+You need to have:
+
+- A Platform.sh account:
+  new users can sign up for a [free trial account](https://auth.api.platform.sh/register)
+- A working project,
+  either started from a [template](../../development/templates.md) 
+  or with your own code pushed to Platform.sh
+- [Git](https://git-scm.com/downloads)
+- The [Platform.sh CLI](../../administration/cli/_index.md)
+
+## 1. Get your code
+
+If you don't have your app code on your computer, download a copy.
+
+1.  Get your project ID by running `platform projects`.
+
+2.  Get the code by running the following command:
+
+    ```bash
+    platform get {{< variable "PROJECT_ID" >}} {{< variable "TARGET_DIRECTORY_NAME" >}}
+    ```
+
+You can now access your code from the project directory on your computer.
+The CLI created a `.platform/local` directory that's excluded from Git. 
+It contains builds and local metadata about your project.
 
 You can now make changes to your project without pushing to Platform.sh each time to test them. 
 Instead, you can locally build your application using the Platform.sh CLI.
 
 Note that if your app contains services, you need to open an SSH tunnel to connect to them.
 For more information, see how to [connect services](../../add-services#2-connect-the-service).
 
-## 2. Build your site locally
+## 2. Connect to services
+
+If your app requires services to run, you have two options for developing locally:
+
+- [Tethered local development](./tethered.md) involves running your app on a local web server
+  but keeping all other services on Platform.sh and connecting to them over an SSH tunnel.
+- [Untethered local development](./untethered.md) involves running your entire site locally,
+  including all services.
+
+Choose the option that works for you and get your services running.
 
-Before you build your site, 
-check that any runtime or tools needed for the build process are available 
-in your local environment.
+## 3. Build your site locally
 
 If you want your local development environment to be enclosed 
 so your main system remains unaffected by the build and its dependencies, 
 you can use a local virtual machine.
 
 To build your site locally:
 
-1. Run the following command:
+1.  Install any dependencies or tools needed for the build process.
 
-   ```bash
-   platform build
-   ```
+2.  Run the following command:
 
-2. When prompted, specify the source directory and the build destination.
+    ```bash
+    platform build
```

Comment:
There is at this time no concept of "build" in the add-on, although there may be in the future. But currently this whole section is misdirected. One can't use `platform build` to do anything useful. Build is done on `ddev start`.

Reply:
This part of the PR is about a page for doing things not with DDEV. The first part of the page should point people in the direction of DDEV if they use PHP and then offer these steps as an alternative.